docs/guides/qedma-qesem.ipynb

```diff
@@ -37,40 +37,56 @@
    "id": "74823696",
```

Comment:
<p><code>default_precision</code> needs to define a default as it says what to do if a precision is not specified by a PUB. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>